### DISCUSSION TITLE
Jetson AGX Orin on Kirkstone

### DISCUSSION THREAD

**Discussion by om26er** [OE4T/meta-tegra#1744] [Q&A] [answered]
We have our Yocto build for kirkstone that we have been using for a few years. I am now working to create an image for Jetson AGX Orin and it looks like that hardware is only supported in newer recipes (master, scarthgap).

What effort would it take to port meta-tegra kirkstone branch to Jetson AGX Orin. Should we rather spend time to upgrade our Yocto image to scarthgap first?

Note: Right now our image works for RaspberryPi and imx8mp platform

**Comment by madisongh** [comment, marked as answer]
As mentioned on [this wiki page](https://github.com/OE4T/meta-tegra/wiki/Which-branch-should-I-use%3F), the kirkstone branch is based off of Jetson Linux 35.6.0, which does have support for AGX Orin.

**Reply by om26er** [reply]
@madisongh thanks for the response. I need to decide now if I should first work on porting out image to scarthgap OR put the effort to port meta-tegra to kirkstone.
